src/pages/[...slug].tsx

```diff
@@ -328,6 +348,26 @@ const ContentPage: NextPage<ContentPageProps> = ({
                 />
             )}
             {!vidOrPod && <DocumentBody content={contentAst} />}
+            <div
+                sx={{
+                    display: [null, 'flex'],
+                    columnGap: [0, 'inc70'],
+                    marginTop: ['inc40'],
```

Comment:
This is still displaying on mobile. Display needs to be `'none'` instead of `null`. Also, the sx responsiveness array is `[mobileSmall, mobileLarge, tablet, desktopSmall, desktopLarge]`. So if you want it to be visible on tablet and invisible on mobile, it should be `display: ['none', null, 'flex']`.